cmd/kaniko-ecr/main.go

```diff
@@ -64,6 +64,11 @@ func main() {
 			Value:  "Dockerfile",
 			EnvVar: "PLUGIN_DOCKERFILE",
 		},
+		cli.StringFlag{
+			Name:   "docker-registry",
+			Usage:  "docker registry",
+			EnvVar: "PLUGIN_DOCKER_REGISTRY,DOCKER_REGISTRY",
```

Comment:
What is the difference between `DOCKER_REGISTRY`, `PLUGIN_REGISTRY`, `PLUGIN_REPO`? It would help if we expanded the usage strings for each of these flags with verbose explanation and sample values.